pkgs/build-support/build-fhs-userenv/chrootenv/chrootenv.c

```diff
@@ -18,17 +18,34 @@
 #include <sys/stat.h>
```

Comment:
You can remove the `#include <ftw.h>` above this (can't review that line unfortunately)